app/src/main/res/values/strings.xml

```diff
@@ -52,5 +61,15 @@
     <string name="contact_already_deleted">It looks like the contact was already deleted.</string>
     <string name="something_went_wrong">Something went wrong</string>
     <string name="something_went_wrong_try_again">There went something wrong deleting this contact, please try again.</string>
-    <string name="no_devices_found">No devices found…</string>
+    <string name="no_devices_found">No devices found...</string>
```

Comment:
Why does this line change in every new pr that has cahnges in this file? I swear I've see it being changed from ... (spaced dots) to … (tight dots) last week?